data-serving/data-service/api/openapi.yaml

```diff
@@ -911,38 +874,22 @@ components:
             properties:
                 query:
                     type: string
-                country:
+                countryISO3:
```

Comment:
If we are using ISO 3 letter codes, then this should be minLength: 3, maxLength: 3, example: GBR

Reply:
Yeah you are right. I think that for now I will just change the variable name to ISO2 just to finish this task as it requires less changes. I will create a new task for switching to ISO3 codes but I think it will require also some changes in the Geocoder